settings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd"
+          xmlns="http://maven.apache.org/SETTINGS/1.1.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <servers>
+        <server>
+            <id>bintray</id>
+            <username>${env.BINTRAY_USER}</username>
+            <password>${env.BINTRAY_KEY}</password>
+        </server>
+    </servers>
+    <profiles>
+        <profile>
+            <id>oss-jfrog-snapshots</id>
+            <repositories>
```

Comment:
This is how it is documented to use the settings.xml with the target repository and the active profile. The repository within the pom is for snapshot dependencies... Hm, but maybe you are right and the target repository is set with the distributionManagment and do not need to be set twice... I will investigate in how to use the settings.xml.